### DISCUSSION TITLE
adding my own photos on faceswapper models

### DISCUSSION THREAD

**Discussion by Bold0077** [community/community#69296] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello guys please i want to add my own photos on the deepfacelive faceswapper , I don't to use the default models , please how can i add my own photos on the face swapper models Deepfacelive

**Comment by queenofcorgis** [comment]
Hi @Bold0077 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Deepface. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
